app/api/comment_routes.py

```diff
@@ -55,13 +60,13 @@ def create_comment(expense_id):
     return {'errors': validation_errors_to_error_messages(form.errors)}, 400
 
 
-@comment_routes.route('/<int:id>', methods=["PUT"])
+@comment_routes.route('/<int:comment_id>', methods=["PUT"])
 @login_required
-def update_comment(id):
+def update_comment(comment_id):
     """
     Updates a comment
     """
-    comment = Comment.query.get(id)
+    comment = Comment.query.get(comment_id)
     # checks if comment exists
     if not comment:
         return {'errors': f"Comment {id} does not exist."}, 400
```

Comment:
@Dmytro-Yakovenko if you are going to change parameters. be more careful to change the old parameter everywhere that it is used